src/dodal/devices/i24/focus_mirrors.py

```diff
@@ -0,0 +1,60 @@
+from ophyd_async.core import StandardReadable, StrictEnum
+from ophyd_async.epics.core import epics_signal_rw
+
+from dodal.common.signal_utils import create_hardware_backed_soft_signal
+
+
+class HFocusMode(StrictEnum):
```

Comment:
It would be useful to document how the focus mirrors work. I'm not sure what these words mean!